src/types/index.ts

```diff
@@ -29,26 +29,45 @@ export type DataCubeCollection = {
   "cube:variables": GenericObject;
 } & StacRender;
 
+
 export type StacCollection = Collection & DataCubeCollection;
 
 export type CollectionConfig = {
   id: string;
   collectionStacUrl: string;
+  stacSearchUrl: string;
   displayName: string;
   tiler: string;
-  datetime_range?: [string, string];
+  timeseriesType?: string;
+  lastAvaliableDatetime?: string;
   stac: StacCollection;
 }
 
 export type LayerConfig = {
   id: string;
   name: string;
   isVisible: boolean;
-  datetime_range?: [string, string];
+  timeseriesType?: string; // forecast or analysis
```

Comment:
You could type this a bit stronger if you want
```suggestion
  timeseriesType?: "forecast" | "analysis"
```